coalib/processes/Processing.py

```diff
@@ -405,13 +405,21 @@ def instantiate_processes(section,
                         'control_queue': control_queue,
                         'timeout': 0.1}
 
+    loaded_local_bears_count = len(local_bear_list)
     local_bear_list[:], global_bear_list[:] = instantiate_bears(
         section,
         local_bear_list,
         global_bear_list,
         complete_file_dict,
         message_queue,
         console_printer=console_printer)
+    loaded_valid_local_bears_count = len(local_bear_list)
+
+    if(loaded_local_bears_count != loaded_valid_local_bears_count):
+        if cache:
+            log_printer.debug("The cached fils have been reset because one of "
+                              'bears did not met one of it\'s requirements ')
```

Comment:
it should be 'one of the bears' instead of 'one of bears' and change 'did not met' to 'did not meet'